corpus/statements.txt

```diff
@@ -461,6 +460,9 @@ def e(**list: str):
 def f():
   nonlocal a
 
+def g(h, i, *, j, k=100, **kwarg):
```

Comment:
@maxbrunsfeld I noticed a quirk in Python's function declaration syntax:

```python
>>> def example(a, *, b, c):
...     print(a, b, c)
...
>>> example("a", b="b", c="c")
a b c
```

If you have a bare `*` in the parameter list, all params after the `*` become keyword args.

It is a syntax error to have the `*` at the end of the parameter list, however.